outputs.tf

```diff
@@ -155,3 +155,13 @@ output "custom_resolver_hub" {
   description = "The custom resolver created for the hub vpc. Only set if enable_hub is set and skip_custom_resolver_hub_creation is false."
   value       = length(ibm_dns_custom_resolver.custom_resolver_hub) == 1 ? ibm_dns_custom_resolver.custom_resolver_hub[0] : null
 }
+
+output "dns_endpoint_gateways_id" {
+  description = "The endpoint gateways in the bound to VPC that are allowed to participate in this DNS resolution binding. Only set if enable_hub is false and enable_hub_vpc_id are true. "
```

Comment:
I appreciate this was taken from the provider, but this sentence makes no sense. The list here are the VPEs that are made available for DNS resolution in the created VPC.